src/main/java/com/amihaiemil/charles/aws/StEsEndPoint.java

```diff
@@ -40,5 +40,13 @@
     public StEsEndPoint(final String name) {
         super(name);
     }
-
+ 
+    @Override
+    public String read() {
+        String endPoint = super.read();
+        if(!endPoint.endsWith("\\")) {
```

Comment:
@SherifWaly We need "/" (slash), not "\" (backslash) :D Sorry, I might have written it wrong when I let the comment the first time.